data-pipeline-ffi/src/trace_exporter.rs

```diff
@@ -923,8 +923,8 @@ mod tests {
             assert_eq!(response.assume_init().body.to_string_lossy(), response_body);
 
             ddog_trace_exporter_free(exporter);
-            // It should receive 3 payloads: app-started, metrics and app-closing.
-            mock_metrics.assert_hits(3);
+            // It should receive 1 payloads: metrics
+            mock_metrics.assert_hits(1);
```

Comment:
/question

do all the libraries have telemetry which send `started` and `closing`?

Just wondering if this would ever make sense to do all the telemetry via native.

Reply:
Yes. This is tested in system-tests
The issue is the additional data sent with app-started and app-closing. The existing telemetry clients in libraries also send their dependencies and configuration.

There is an API for this on the telemetry client, and we could eventually replace the in-language telemetry client with the libdatadog one, but IMO this should not be done in the same initiative as the trace exporter integration to prevent scope creep.